homeassistant/components/letpot/config_flow.py

```diff
@@ -51,11 +59,20 @@ class LetPotConfigFlow(ConfigFlow, domain=DOMAIN):
 
     async def _async_validate_credentials(
         self, email: str, password: str
-    ) -> dict[str, Any]:
+    ) -> tuple[dict[str, str], dict[str, Any] | None]:
+        """Try logging in to the LetPot account and returns credential info."""
         websession = async_get_clientsession(self.hass)
-        client = LetPotClient(websession)
-        auth = await client.login(email, password)
-        return {
+        try:
+            client = LetPotClient(websession)
```

Comment:
Does this need to be within the `try` block?

Reply:
Strictly speaking not, only the login call should throw exceptions. Updated.